app/js/components/EscrowList.jsx

```diff
@@ -67,7 +68,7 @@ const EscrowList = (props) => (
             <td>{getEscrowStateText(escrow, props.t)}</td>
             <td>{escrow.buyer === web3.eth.defaultAccount ? escrow.seller : escrow.buyer}</td>
             <td>{escrow.amount}</td>
-            <td>{new Date(escrow.expirationTime * 1000).toString()}</td>
+            <td>{moment(escrow.expirationTime * 1000).toString()}</td>
```

Comment:
I can see we are doing this multiple times, do you think we should move this conversion to the selector?